helm/ngrok-operator/templates/agent/deployment.yaml

```diff
@@ -129,7 +129,7 @@ spec:
           initialDelaySeconds: 5
           periodSeconds: 10
         resources:
-        {{- toYaml .Values.resources | nindent 10 }}
+        {{- toYaml $agent.resources | nindent 10 }}
```

Comment:
nit: you might consider using the original value as a fallback. Just thinking this is passive for the helm values since they both default to empty. But in our case, we set this value so its being applied to both the manager and agent. This change would remove any resource requests from our agent unless we explicitly add the agent resources.

Reply:
Yeah, I called that out in breaking changes. I don't think we should default to the root, but to maintain passivity I guess we can. This is something I want to fix by 1.0. We should have separate configuration for each of the 3 workloads, and then globals that apply to all.

Reply:
Talked about this more synchronously, if we make this passive, there is no way to only set resource requests on the api-manager pods, since falling back to those values for the agent and bindings-fowarder if they are empty would overwrite that.

I think we are just going to note this in the changelog as mentioned on the PR.